mercury/tests/test_models.py

```diff
@@ -0,0 +1,72 @@
+from django.test import TestCase
+import datetime
+from mercury.models import (
+    SimulatedData,
+    TemperatureSensor,
+    AccelerationSensor,
+    WheelSpeedSensor,
+    SuspensionSensor,
+    FuelLevelSensor,
+)
+
+# Test Values
+TEST_TEMP = 999.0
+TEST_ACCEL_Y = 9.81
+TEST_WHEEL_SPEED_FR = 30
+TEST_SUSPENSION_FR = 2
+TEST_FUEL = 6
+
+
+def create_simulated_data_objects():
+    SimulatedData.objects.create(
+        temperature=TEST_TEMP, created_at=datetime.datetime.now()
+    )
+    TemperatureSensor.objects.create(
+        temperature=TEST_TEMP, created_at=datetime.datetime.now()
+    )
+    AccelerationSensor.objects.create(
+        acceleration_x=TEST_ACCEL_Y, created_at=datetime.datetime.now()
+    )
+    WheelSpeedSensor.objects.create(
+        wheel_speed_fr=TEST_WHEEL_SPEED_FR, created_at=datetime.datetime.now()
+    )
+    SuspensionSensor.objects.create(
+        suspension_fr=TEST_SUSPENSION_FR, created_at=datetime.datetime.now()
+    )
+    FuelLevelSensor.objects.create(
+        current_fuel_level=TEST_FUEL, created_at=datetime.datetime.now()
+    )
+
+
+class TestSimulatedData(TestCase):
```

Comment:
Since this is testing more than `SimulatedData`, either move the other tests to a new class, or change the name of this to be more generic for now, such as `TestSensorModels`.